### DISCUSSION TITLE
Smaller screen

### DISCUSSION THREAD

**Discussion by dpwe** [shorepine/tulipcc#126] [Ideas]
The 10.1" screen is .. bigger than some screens you could imagine.

I'd like a tulip as a single brick with 4 or 5" display on top. 

Seems like something like [this](https://www.crystalfontz.com/product/cfaf800480h0043sc-800x480-4-point-3-inch-capacitive-touchscreen-tft-display) (4.3", 800x480 pixels, capacitative touch-screen) should be possible to attach, with some software accommodation?  Annoyingly, although it uses 40 pin FPC for the display, it needs a second 10 pin FPC for the I2C interface to the touch sensor.

**Comment by bwhitman** [comment]
I would love that form factor, too. And the big resolution of Tulip definitely makes a lot of things harder than they should be! 

One thing to look at is non-TTLRGB screens at that size. Like SPI. I believe there to be many, many screens of smaller size that that the ESP32S3 natively supports that work with embedded display controllers. 

As long as it works on `esp_lcd`, it should work on Tulip! ( eventually.)

https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/api-reference/peripherals/lcd.html

**Comment by bwhitman** [comment]
Here's a 7" with the same pinout as our 10.1" https://www.hotmcu.com/7-inch-800x480-tft-lcd-display-with-capacitive-touch-panel-p-216.html

**Comment by nvillar** [comment]
I think this 5" one is also compatible, right? It looks to be the smallest one of the range. Really cute!

https://www.hotmcu.com/5-inch-800x480-tft-lcd-display-with-capacitive-touch-panel-p-217.html

**Comment by bwhitman** [comment]
Neat! Yes, that would work as a drop in to Tulip with no changes other than the `H_RES` and `V_RES` in `display.h`. I'm going to get one, looks adorable
